internal/encryption/keyprovider/gcp_kms/config.go

```diff
@@ -16,6 +16,11 @@ import (
 	kms "cloud.google.com/go/kms/apiv1"
 )
 
+// Can be overridden for test mocking
+var newKeyManagementClient func(ctx context.Context, opts ...option.ClientOption) (keyManagementClient, error) = func(ctx context.Context, opts ...option.ClientOption) (keyManagementClient, error) {
```

Comment:
That line is... very hard to read. I'd either type alias or break it into multiple lines.